### PR TITLE
added filenames

### PR DESCRIPTION
**Pull request by FaithMo**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by Ngoakor12 — COMMENTED** [review]
Hey Faith, just have a few comments.

- I think the placing of the environment section is in an odd place(top of the project before the project intro paragraphs), would the section coming before "Write some functionality" section not be slightly better for the flow of the project?
- The project instructions says to add the `data.json` file to your repo, so maybe add it in the src folder too?
- The project instructions mention nothing about tests nor do the tags mention it anywhere, If they are not required, the jasmine stuff might not be necessary. If they are, I think the tags or instructions should mention it somewhere.

Side note: I have completed this project on my own without being reviewed so take my suggestions with a pinch of salt as I'm not sure.

**Comment by FaithMo** [comment]
Hey, yeah I forgot the project doesn't need the tests. 
I followed how other projects are set up, they put the file and directory naming at the top of the page.

File: content/projects/understanding-loops/_index.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ tags:
...
+
+### Javascript
+ 
+Your directory structure should look like this.
+ 
+```
+   > src
+       - shopping_cart.js
+       - data.json
+   - package.json
```
Comment (Ngoakor12) [inline content/projects/understanding-loops/_index.md]: Is the package.json required if there are no packages to install?

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
The placing of the section is okay if it's consistent with other projects.

**Comment by FaithMo** [comment]
Hi @Ngoakor12 , I have actually removed the data.json file because when i take a close look at the instructions, they say that the learner's code needs to work for data that has a similar structure to data.json, so that means they do not need to have the exact same database but that is similar, so we do not need to specify data.json in the directory.

**Review by Ngoakor12 — APPROVED** [review]
looks good to me

**Review by Sbonelo01 — APPROVED** [review]

**Review by Vuyisanani — APPROVED** [review]

**Review by ry-oc — APPROVED** [review]
looks fine

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**